### PR TITLE
ENG-142: Fix 'Meet The Team' Component

### PR DESCRIPTION
**Pull request by rfougy**
# Summary
- Styling changes made for the `MeetTheTeam` component, including responsive font sizes and layouts.
- Animation transitions upon clicking across `TeamMember` components achieved.
- Breakpoints added to `tailwind.config` file.

[Link to Linear Ticket](https://linear.app/mun-e/issue/ENG-142/fix-meet-the-team-component-and-enhance-thumbnail-image-quality-to)

# Other Changes
- Removed `BiosOverlay` component from 'Meet The Team' section. Conversations were had with Caleb, and the decision was made to remove the modal. In the near future, Caleb intends to create a section at the bottom of the descriptions for "industry experience" and additional info the team would like to share.
- Minor changes made to the `Mission` component in order to resolve spacing for the 'Meat The Team' section.

### CODE REVIEW

File: src/constants/bios.ts
Comment (rfougy) [inline src/constants/bios.ts:1]: Majority changes (except for adding Ashley as team member and new images) due to prettier formatting.

Comment (HoseaCodes) [reply src/constants/bios.ts:1]: You can remove any reference to Ashley.

File: src/assets/team-members/Ashley-Lewis.png
Comment (HoseaCodes) [inline src/assets/team-members/Ashley-Lewis.png:1]: You can remove this image

File: src/components/meet-the-team/ExpandedTeamCard.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { MemberProps } from '../../interfaces/teamMembers';
+
+export const ExpandedTeamCard: React.FC<{
+  teamMembers: MemberProps[];
+  index: number;
+  fadeAnime: boolean;
+}> = ({ teamMembers, index, fadeAnime }) => {
+  const truncateAfterFirstSentence = (
```
Comment (HoseaCodes) [inline src/components/meet-the-team/ExpandedTeamCard.tsx]: Let’s move this into utilities functions

**Review by HoseaCodes — CHANGES_REQUESTED** [review]

File: src/assets/team-members/Ashley-Lewis.png
Comment (rfougy) [reply src/assets/team-members/Ashley-Lewis.png:1]: Ok. This image is also include in the Contact page, for now I've replaced the image of Ashley with Dana. I'll let Caleb know of this change and see if he prefers a different member photo.

**Review by HoseaCodes — DISMISSED** [review]

**Review by HoseaCodes — APPROVED** [review]

**Pull request merged**